packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1379,6 +1382,101 @@ The following conditions may be applied to Password Policy:
   },
 ```
 
+<ApiLifecycle access="ie" /><br>
+
+With Identity Engine, Recovery Factors can be specified inside the Password Policy Rule object instead of in the Policy Settings object. Recovery factors on the rule is defined inside the `selfServicePasswordReset` Action.
```

Comment:
```suggestion
With Identity Engine, Recovery Factors can be specified inside the Password Policy Rule object instead of in the Policy Settings object. Recovery Factors for the Rule are defined inside the `selfServicePasswordReset` Action.
```